### PR TITLE
added grid component

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**

### CODE REVIEW

File: components/grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        &lt;div class="grid-items"&gt;Grid 3&lt;/div&gt;
+        &lt;/div&gt;
+            </code>
+        </pre>
+    </div>
+
+<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/10.6.0/highlight.min.js"></script>
+            <script>
+                hljs.highlightAll();
+            </script>
```
Comment (smkmallik) [inline components/grid/grid.html:91]: Just a minor change here, fix the formatting here, add a tab to fix the spacing.

File: components/grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                </div>
+            </div>
+            <pre>
+      <code class="html">
+          &lt;div class="grid__border"&gt;
+          &lt;div class="grid__container grid-two"&gt;
+          &lt;div class="grid-items"&gt;Grid 1&lt;/div&gt;
+          &lt;div class="grid-items"&gt;Grid 2&lt;/div&gt;
+          &lt;/div&gt;
+      </code>
```
Comment (smkmallik) [inline components/grid/grid.html:61]: Add a tab to align the space

File: components/grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    padding: 1rem 0rem;
+    box-shadow: 0px 0px 13px -7.5px var(--dark-color);
+    padding: 1rem;
+}
+
+.grid__border {
+    border: 3px solid var(--paragraph-text-color);
+    width: 100%;
+    padding: 1rem
+}
```
Comment (smkmallik) [inline components/grid/grid.css:21]: add global declarations for all the direct value declarations

**Comment by smkmallik** [comment]
Grid components changes seems good. The HTML structure and the CSS styling looks okay. 

Just a small note: 
 - I think you can try to write a bit of semantic HTML5 code now.  
 - All the direct CSS declarations can be declared globally. 
 - The formatting of file should be aligned properly.

Other than that the changes are good to merge.

Keep this up 🔥

**Review by smkmallik — APPROVED** [review]
I have added some comments. Check it out and later you can further improve it.

**Review by MurtazaBagwala07 — APPROVED** [review]
* Code structure looks clean
* Grid component looks code is clean
* Just a suggestion , you can add more grid styles

Rest looks good to merge

**Comment by vivekgudipudi** [comment]
- good naming convention
- if possible try to use more utils
- UI looks great

**Pull request merged**